Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -984,7 +984,11 @@ private void InitializeAllServices()
         private void ResetAllServices()
         {
             //If the Mixed Reality Toolkit is not configured, stop.
-            if (activeProfile == null) { return; }
+            if (ActiveProfile == null)
+            {
+                DebugUtilities.LogCriticalError("Unable to reset services. There is no active profile configured.");
```

Comment:
Seems like a lot of duplicated code, couldn't we just log this critical error in the `ActiveProfile` property?

Reply:
> Seems like a lot of duplicated code, couldn't we just log this critical error in the ActiveProfile property?

That sounds like a good optimization. This change was to match the pattern used in several other methods (ex: GetService).

Reply:
Updated ActiveProfile to always log a critical error if there isn't one configured